.circleci/unittest/linux/scripts/setup_env.sh

```diff
@@ -38,7 +38,9 @@ conda activate "${env_dir}"
 printf "* Installing dependencies (except PyTorch)\n"
 conda env update --file "${this_dir}/environment.yml" --prune
 if [ "${os}" == Linux ] ; then
-    pip install clang-format
+    clangformat_path="${root_dir}/clang-format"
```

Comment:
As far as I understand other libraries like conda are put directly in root. So I do the same with clang-format.